test/test_additional_import.py

```diff
@@ -162,8 +135,9 @@ def test_missing_columns1(single_import_parameters):
         )
 
 
-def test_missing_columns2(single_import_parameters, caplog):
-    dataset, original, excel, concept_name = single_import_parameters
+# TODO: Discuss with Gereon, Test has multiple asserts
```

Comment:
Okay, I got up to here, I'll look at this in more details

Reply:
There are multiple asserts in several tests. So probably it's easiest if we have a meeting where we go through all the test files together and have a look at the comments left.